CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Platform Version 0.16.0 - 2025-02-26
```

Comment:
v0.16.1?

Reply:
good catch!

Reply:
I think that git cliff was created to run after the tag release is created.

Reply:
Maybe we can automatize it

Reply:
I haven't verified, but maybe: `git cliff --tag $(cat VERSION)`